hw2/04-background.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04 - Background</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <script defer src="04-background.js"></script>
+  </head>
+  <body id="body-color">
+    <main>
+      <div class="bootstrap-grid" id="bootstrap-grid">
+        <div class="row">
+          <div class="container col-12 col-sm-7 col-md-6 col-lg-5 rounded bg-white" id="color-change-card">
+              <h1 style="margin-bottom: 20px; margin-top: 10px;">Exercise 04 - Background</h1>
```

Comment:
Use bootstrap classes

Reply:
Same as previous comment. I used bootstrap classes elsewhere and the assignment doesn't specify that all styling must be done in bootstrap.